tools/ci_build/github/linux/ort_minimal/build_minimal_ort_android_baseline_and_report_bin_size.sh

```diff
@@ -33,12 +34,18 @@ python3 -m pip install --user mysql-connector-python
 
 # Post the binary size info to ort mysql DB
 # The report script's DB connection failure will not fail the pipeline
-python3 /onnxruntime_src/tools/ci_build/github/windows/post_binary_sizes_to_dashboard.py \
-    --ignore_db_error \
-    --commit_hash=$BUILD_SOURCEVERSION \
-    --size_data_file=/build/MinSizeRel/binary_size_data.txt \
-    --build_project=onnxruntime \
-    --build_id=$BUILD_ID
+if [ $BUILD_REASON="IndividualCI" ] OR [ $BUILD_REASON="BatchedCI" ]; then
```

Comment:
Not sure this is correct.
https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=265960&view=logs&j=91d18614-091d-543c-f7ca-c2dad642e5d4&t=e3f80ac6-71a1-5453-3040-e0cf50d179c3&l=951
+ '[' PullRequest=IndividualCI ']' OR '[' PullRequest=BatchedCI ']'
/onnxruntime_src/tools/ci_build/github/linux/ort_minimal/build_minimal_ort_android_baseline_and_report_bin_size.sh: line 37: [: too many arguments

Reply:
Corrected